### PR TITLE
Adding the lookup files to the repository

### PR DESCRIPTION
**Pull request by berroar**
This PR has been raised to add the current lookup files to the repository and update the readme.

### CODE REVIEW

**Comment by berroar** [comment]
[Create repo to manage socio-cultural lookup json (S)](https://trello.com/c/PWnzz6FQ/3688-create-repo-to-manage-socio-cultural-lookup-json-s)

**Comment by berroar** [comment]
[Create repo to manage socio-cultural lookup json (S)](https://trello.com/c/PWnzz6FQ/3688-create-repo-to-manage-socio-cultural-lookup-json-s)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
 # eq-lookup-suggestions-data
 eQ Lookup Suggestions Data
+
+This repository contains the data-sets required for lookup questions in eq-questionnaire-schemas
```
Comment (iwootten) [inline README.md]:
```suggestion
This repository contains versioned TextField suggestion data for eq-questionnaire-runner
```

**Review by ajmaddaford — COMMENTED** [review]
The files shouldn't be in a versioned folder. We will control versioning via Github releases.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
 # eq-lookup-suggestions-data
 eQ Lookup Suggestions Data
+
```
Comment (ajmaddaford) [inline README.md:2]: The heading should be "EQ Lookup Suggestions Data" rather than the repo name.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
+This repository contains versioned TextField suggestion data for eq-questionnaire-runner
+
+| Datasets |       |
```
Comment (ajmaddaford) [inline README.md]: - Datasets -> Dataset
- The second column should have a heading of "Description"

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Review by iwootten — APPROVED** [review]

**Pull request merged**